_routes.php

```diff
@@ -1456,15 +1503,492 @@ function ($request, $response, $args) {
+
+$this->post(
+    __('/object', 'objectslend_routes') . __('/dotake_lend', 'objectslend_routes'),
+    function ($request, $response, $args) {
+        $ok=true;
+            $adherent_id=$_POST['adherent_id'];
+            $payment_type=intval($_POST['payment_type']);
+            $date_begin=$_POST['date_begin'];
+            $date_forecast=$_POST['date_forecast'];
+            $status_id=$_POST['status_id'];
+            $comments=$_POST['comments'];
+            $comments=$_POST['comments'];
+
+        $ct = new ContributionsTypes($this->zdb);
+        $contributions_types = $ct->getList($args['type'] === 'fee');
+        $lendsprefs = new Preferences($this->zdb);
+        foreach ($_POST['ids'] as $object_id) {
+            $object = new LendObject($this->zdb, $this->plugins, (int)$object_id, true);
+            $adherent_id=$_POST['adherent_id'];
+            // ajouter test si objet pris, alors le clore
+            $close=LendRent::closeAllRentsForObject($object_id, "Take by : " . $_POST['adherent_id']);
+            $new=LendRent::newRentForObject($object_id, $date_begin, $date_forecast, $status_id, $adherent_id, $comments);
+            if ($new == false) {
+                $ok=false;
+            }
+
+            // Récupération du prix de location
+            $rentprice = str_replace(',', '.', $object->rent_price);
+            // Ajout d'une contribution
+            if ($rentprice > 0 && $lendsprefs->{Preferences::PARAM_AUTO_GENERATE_CONTRIBUTION}) {
+                $contrib = new Contribution($this->zdb, $this->login);
+                $info = str_replace(array('{NAME}', '{DESCRIPTION}', '{SERIAL_NUMBER}', '{PRICE}', '{RENT_PRICE}', '{WEIGHT}', '{DIMENSION}'), array($object->name, $object->description, $object->serial_number, $object->price, $object->rent_price, $object->weight, $object->dimension), $lendsprefs->{Preferences::PARAM_GENERATED_CONTRIB_INFO_TEXT});
+                $values = array(
+                    'id_adh' => $adherent_id ,
+                    'id_type_cotis' =>  "4" ,
+                    'montant_cotis' => "10",
+                    'contribution_type' => $lendsprefs->{Preferences::PARAM_GENERATED_CONTRIBUTION_TYPE_ID},
+                    'date_enreg' => date(_T("Y-m-d")),
+                    'date_debut_cotis' => date(_T("Y-m-d")),
+                    //'date_fin_cotis' => date(_T("Y-m-d")),
+                    'type_paiement_cotis' => $payment_type,
+                    'info_cotis' => $info
+
+                );
+                $contrib->check($values, array(), array());
+                $contrib->store();
+            }
+        }
+
+        $id_adh=intval($_POST['adherent_id']);
+        $member = new Adherent($this->zdb, $id_adh);
+
+
+
+        if ($ok === false) {
+            $error_detected =
+                _T("Error")
+                . " : <b>" . $member->name . " " . $member->surname . " " . $object->name . " " . $_POST['date_begin'] . " " . $_POST['date_forecast'] . "</b>";
```

Comment:
```suggestion
                . " : " . $member->name . " " . $member->surname . " " . $object->name . " " . $_POST['date_begin'] . " " . $_POST['date_forecast'] ;
```

`<b>` est une balise destinée à l'affichage, et de toutes façons, il n'y a pas de raison que c emessage ne soit pas affiché comme les autres :)